core/src/drawable.rs

```diff
@@ -65,8 +66,25 @@ pub trait Drawable {
     /// The pixel color type.
     type Color: PixelColor;
 
+    /// The return type of the [`draw`] method.
+    ///
+    /// The `Output` type can be used to return results and values produced from the drawing of the
+    /// current item. For example, rendering two differently styled text items next to each other
+    /// can make use of a returned value, allowing the next text to be positioned after the first:
+    ///
+    /// ```rust,ignore
```

Comment:
Should we add a TODO comment to remember to enable this when the `Text` `Drawable` impl is updated?